frontend/src/Components/NewRunConfig/AdvancedSettings/AdvancedSettings.tsx

```diff
@@ -13,88 +16,33 @@ interface AdvancedSettingsProps {
 export default function AdvancedSettings(props: AdvancedSettingsProps) {
   const { config, setConfig } = props;
 
-  const [disableTotalFood, setDisableTotalFood] = useState(Boolean);
-
   function configHandler<Key extends keyof SimConfig>(value: any, keyString: any) {
     var key: Key = keyString; // converting keyString to type Key
     config[key] = value;
     setConfig(config);
+    console.log(config);
```

Comment:
Could keep could remove? Could be useful when things crash

Reply:
Will remove.